core/domain/exp_domain_test.py

```diff
@@ -8384,21 +8384,21 @@ def test_create_html_math_rich_text_info(self):
             exploration_math_rich_text_info.to_dict(), {
                 'exp_id': 'exp_id1',
                 'math_images_generation_required': True,
-                'latex_values_without_svg': ['abc', 'x']
+                'latex_strings_without_svg': ['abc', 'x']
             })
 
-    def test_validate_when_latex_values_not_list(self):
+    def test_validate_when_latex_strings_not_list(self):
         with self.assertRaisesRegexp(
             Exception,
-            'Expected latex_values to be a list, received '
+            'Expected latex_strings to be a list, received '
             'invalid_latex_format'):
             exp_domain.ExplorationMathRichTextInfo(
                 'exp_id1', True, 'invalid_latex_format')
 
-    def test_validate_when_each_latex_values_are_string(self):
+    def test_validate_when_each_latex_strings_are_string(self):
```

Comment:
Isn't this test testing the case when they are not strings?

Reply:
done